events/wc_cc_missions.txt

```diff
@@ -40,6 +40,17 @@ character_event = {
 				modifier = { factor = 0 is_playable = no } # Only playable characters get focuses
 				set_character_flag = cenarion_circle_mission_theology
 			}
+			10 = {
+				modifier = { factor = 0 is_landed = no }
+				modifier = { factor = 0 has_character_flag = barrow_den_completed }
+				
+				set_character_flag = cenarion_circle_mission_barrow_den
+			}
+			10 = {
+				modifier = { factor = 0 is_landed = no }
+				modifier = { factor = 0 any_demesne_province = { has_hospital = yes } }
```

Comment:
You will never be given the mission to build a hospital if you already have at least one. Is it WAD?